docs/tracing/03-02-Tracing.md

```diff
@@ -18,7 +18,7 @@ The Envoy sidecar uses Jaeger to trace the request flow in the Istio Service Mes
 
 For details, see [Istio's Distributed Tracing](https://istio.io/docs/tasks/observability/distributed-tracing/).
 
-## Install Jaeger locally
+## Install Tracing locally
```

Comment:
"Tracing" is only the name of the chart. But the functionality is provided by jaeger and you are accessing the jaeger UI.
I think we should keep it for now and re-write the whole page later as it can be improved indeed

Reply:
Ok, then let's have a separate task for that - I will create it.